FAIL: TestAcceptJob

One of the tests failed.
See https://github.com/playmeme/agentmarket/actions/runs/23457823689/job/68251585324 

```
=== CONT  TestSignupMissingFields/missing_email
2026/03/23 20:11:43 "POST http:// HTTP/1.1" from  - 400 64B in 15.449µs
2026/03/23 20:11:43 INFO job accepted, moved to SOW_NEGOTIATION request_id=cfe5cbe700bc201c handler=accept_job job_id=62ab6ffe-3138-4a37-89d8-38b6b644dcfe agent_id=a589e66a-f155-4fae-9508-f2d6dfca4f17 sow_id=d6c2c163-5e92-4951-bd98-857eab9c5c72
2026/03/23 20:11:43 "POST http:// HTTP/1.1" from  - 200 322B in 767.813µs
=== NAME  TestAcceptJob
    jobs_test.go:154: expected status IN_PROGRESS, got "SOW_NEGOTIATION"
=== CONT  TestSignupMissingFields/invalid_role
--- FAIL: TestAcceptJob (0.01s)
=== CONT  TestSignupMissingFields/missing_password
2026/03/23 20:11:43 "POST http:// HTTP/1.1" from  - 400 50B in 14.217µs
2026/03/23 20:11:43 "POST http:// HTTP/1.1" from  - 400 64B in 38.772µs
--- PASS: TestSignupMissingFields (0.01s)
    --- PASS: TestSignupMissingFields/missing_email (0.00s)
    --- PASS: TestSignupMissingFields/invalid_role (0.00s)
    --- PASS: TestSignupMissingFields/missing_password (0.00s)
2026/03/23 20:11:43 INFO migrations complete
2026/03/23 20:11:43 WARN authz failure: hire agent requires EMPLOYER role request_id=f4f2122575a6abab handler=hire_agent role=AGENT_HANDLER
2026/03/23 20:11:43 "POST http:// HTTP/1.1" from  - 403 46B in 52.067µs
--- PASS: TestHireAgentWrongRole (0.01s)
2026/03/23 20:11:43 ERROR email send failed: provider error to=alice@example.com subject="Verify your AgentMarket email" status=401
2026/03/23 20:11:43 WARN verification email failed to send request_id=7ba02a66b791368f handler=signup user_id=b1ef8a0e-9aed-4cde-b9b1-df68b265e601 email=alice@example.com error="resend API returned status 401"
2026/03/23 20:11:43 "POST http:// HTTP/1.1" from  - 201 401B in 180.687036ms
--- PASS: TestSignup (0.19s)
FAIL
FAIL	agentmarket	0.488s
FAIL
Error: Process completed with exit code 1.
```

--- a/backend/jobs_test.go
+++ b/backend/jobs_test.go
@@ -150,8 +150,8 @@ func TestAcceptJob(t *testing.T) {
 
 	var updated Job
 	json.Unmarshal(rr.Body.Bytes(), &updated)
-	if updated.Status != "IN_PROGRESS" {
-		t.Errorf("expected status IN_PROGRESS, got %q", updated.Status)
+	if updated.Status != "SOW_NEGOTIATION" {
+		t.Errorf("expected status SOW_NEGOTIATION, got %q", updated.Status)
 	}
 }
 

